Missing uninstall target: "make: *** No rule to make target 'uninstall'.  Stop."

Followed the guide:
https://github.com/KhronosGroup/Vulkan-ExtensionLayer/blob/main/BUILD.md#installed-files
I used a `Release` build instead of `Debug` build.
```
sudo make uninstall
make: *** No rule to make target 'uninstall'.  Stop.
```

`Install` works fine.

--- a/BUILD.md
+++ b/BUILD.md
@@ -70,9 +70,6 @@ indicated by *install_dir*:
 - *install_dir*`/share/vulkan/explicit_layer.d` : The Vulkan Extension layer
   JSON files (Linux and MacOS)
 
-The `uninstall` target can be used to remove the above files from the install
-directory.
-
 ## Repository Set-Up
 
 ### Display Drivers
@@ -623,12 +620,6 @@ further customize your installation.
 Also see the `LoaderAndLayerInterface` document in the `loader` folder of the
 Vulkan-Loader repository for more information about loader and layer
 operation.
-
-#### Linux Uninstall
-
-To uninstall the files from the system directories, you can execute:
-
-    sudo make uninstall
 
 #### Linux Tests
 

